### ISSUE TITLE
Responsivni stylovani Pagination

### ISSUE THREAD

**Issue by stepiczekk** [riganti/dotvvm#647]
Control Pagination by mohl dostavat pro kazdy LI nejakou tridu se vzdalenosti od aktivniho LI
<ul>
  <li class="offset-2"> ..</li>
  <li class="offset-1"> ..</li>
  <li class="active"> ..</li>
  <li class="offset1"> ..</li>
  <li class="offset2"> ..</li>
</ul>

dalo by se to pak lepe responsivne stylovat ...

**Comment by exyi** [comment]
@stepiczekk You're right, it would be generally nice if the controls inside the DataPager were a bit more configurable.
